server/util.go

```diff
@@ -213,17 +213,13 @@ func ExternalIP() (string, error) {
 // TrapSignal traps SIGINT and SIGTERM and terminates the server correctly.
 func TrapSignal(cleanupFunc func()) {
 	sigs := make(chan os.Signal, 1)
-	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
+	signal.Notify(sigs, os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
Oh this should just be `os.Interrupt` and `os.Kill` https://golang.org/pkg/os/#Signal